bindings/java/src/main/native/Makefile.am

```diff
@@ -11,28 +11,21 @@ javadir=$(top_srcdir)/bindings/java
 MVNCMD=$(MVN) -B -T 1C -f $(topdir)/bindings/java/pom.xml -Dmaven.repo.local=$(java_build_dir)/.deps \
               -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
 
-BUILT_SOURCES = org_openucx_jucx_ucp_UcpConstants.h \
-                org_openucx_jucx_ucp_UcpContext.h \
-                org_openucx_jucx_ucp_UcpEndpoint.h \
-                org_openucx_jucx_ucp_UcpWorker.h \
-                org_openucx_jucx_ucs_UcsConstants.h \
-                org_openucx_jucx_ucp_UcpListener.h
-
-DISTCLEANFILES = org_openucx_jucx_ucp_UcpConstants.h \
-                 org_openucx_jucx_ucp_UcpContext.h \
-                 org_openucx_jucx_ucp_UcpEndpoint.h \
-                 org_openucx_jucx_ucp_UcpWorker.h \
-                 org_openucx_jucx_ucs_UcsConstants.h \
-                 org_openucx_jucx_ucp_UcpListener.h
-
-org_openucx_jucx_ucp_UcpListener.h:
-org_openucx_jucx_ucp_UcpConstants.h:
-org_openucx_jucx_ucp_UcpEndpoint.h:
-org_openucx_jucx_ucp_UcpWorker.h:
-org_openucx_jucx_ucs_UcsConstants.h:
-org_openucx_jucx_ucp_UcpContext.h:
+JUCX_GENERATED_H_FILES = org_openucx_jucx_*.h
+
+BUILT_SOURCES = $(JUCX_GENERATED_H_FILES)
+
+DISTCLEANFILES = $(JUCX_GENERATED_H_FILES)
```

Comment:
since it's already in MOSTLYCLEANFILES , probably can remove from DISTCLEANFILES